include/SFML/Audio/Music.hpp

```diff
@@ -189,11 +189,11 @@ class SFML_AUDIO_API Music : public SoundStream
     TimeSpan getLoopPoints() const;
 
     ////////////////////////////////////////////////////////////
-    /// \brief Sets the beginning and end of the sound's looping sequence using sf::Time
+    /// \brief Sets the beginning and duration of the sound's looping sequence using sf::Time
     ///
-    /// Loop points allow one to specify a pair of positions such that, when the music
+    /// Loop points allow one to specify the beginning offset and the duration of the loop such that, when the music
```

Comment:
Unrelated, but the `one` breaks the reading flow slightly: `Loop points allow to specify ...`

Make sure to wrap the line so it better fits the surrounding blocks.

Reply:
I think "Loop points allow for specifying" would sound good

Reply:
> Unrelated, but the `one` breaks the reading flow slightly: `Loop points allow to specify ...`
> 
> Make sure to wrap the line so it better fits the surrounding blocks.

Done  ;-;